platforms/hyperledger-fabric/configuration/roles/create/cli_pod/tasks/main.yaml

```diff
@@ -0,0 +1,45 @@
+#############################################################################################
+# This role creates value file for Cli pods
+############################################################################################
+#                      CREATE CLI POD                                                      #
+############################################################################################
+#
+############################################################################################
+# This task creates the value file for the cli pod as per requirements mentioned in 
+# network.yaml
+- name: "Create Value file for CLI Pod"
+  include_role:
+    name: helm_component
+  vars:
+    name: "cli"
+    component_name: "{{ org.name | lower}}-cli"
+    orderer_image: "hyperledger/fabric-orderer:{{ network.version }}"
+    alpine_image: "{{ docker_url }}/alpine-utils:1.0"  
+    peer: "{{ org.services.peers | first }}"
+    orderer: "{{ network.orderers | first }}"
+    component_ns: "{{ org.name | lower}}-net"
+    git_url: "{{ org.gitops.git_ssh }}"
+    git_branch: "{{ org.gitops.branch }}"
+    charts_dir: "{{ org.gitops.chart_source }}"
+    vault: "{{ org.vault }}"
+    fabrictools_image: "hyperledger/fabric-tools:{{ network.version }}"
+    alpine_image: "{{ network.docker.url }}/alpine-utils:1.0"
+    storage_class: "{{ org.name | lower }}sc"
+    values_dir: "{{playbook_dir}}/../../../{{org.gitops.release_dir}}/{{ org.name | lower }}"
+    type: "cli"
+  when: network.organizations is defined and org.cli == "enabled" and org.type == "peer"
```

Comment:
Same comment as above 
(network.organizations and org.cli are wrong checks)
CLI should be up based on the cli tag status of peers, not organizations